frame/contracts/src/lib.rs

```diff
@@ -92,6 +92,7 @@ mod schedule;
 mod storage;
 mod wasm;
 
+pub use wasm::api_doc;
```

Comment:
Please group with the rest of the public exports further down and not with the module declarations.